testproject/Packages/manifest.json

```diff
@@ -1,21 +1,21 @@
 {
   "dependencies": {
-    "com.unity.collab-proxy": "1.9.0",
-    "com.unity.ide.rider": "3.0.7",
-    "com.unity.ide.visualstudio": "2.0.11",
-    "com.unity.ide.vscode": "1.2.3",
-    "com.unity.mathematics": "1.2.1",
+    "com.unity.collab-proxy": "1.15.12",
```

Comment:
Did you want to include this?

Reply:
I initially just needed to update com.unity.transport but took the opportunity to move everything up to the most recent version.